en.html

```diff
@@ -0,0 +1,472 @@
+	<link type="text/css" rel="stylesheet" href="css/languages.min.css" />
+
+	<!-- HTML5 shim and Respond.js for IE8 support of HTML5 elements and media queries -->
+	<!-- WARNING: Respond.js doesn't work if you view the page via file:// -->
+	<!--[if lt IE 9]>
+		<script src="https://oss.maxcdn.com/html5shiv/3.7.3/html5shiv.min.js"></script>
+		<script src="https://oss.maxcdn.com/respond/1.4.2/respond.min.js"></script>
+	<![endif]-->
+</head>
+
+<body>
+	<header id="home">
+		<div class="bg-img" style="background-image: url('./img/mickoflus-title.jpg');">
+			<div class="overlay"></div>
+		</div>
+		<nav id="nav" class="navbar nav-transparent">
+			<div class="container">
+
+				<div class="navbar-header">
+					<div class="navbar-brand">
+						<a href="index.html">
+							<img class="logo" src="img/logo.svg" alt="logo">
+							<img class="logo-alt" src="img/logo-alt.svg" alt="logo">
+						</a>
+					</div>
+
+					<div class="nav-collapse">
+						<span></span>
+					</div>
+				</div>
+
+				<ul class="main-nav nav navbar-nav navbar-right">
+					<li><a href="#home">Introduction</a></li>
+					<li><a href="#about">About Míčkoflusu</a></li>
+					<li><a href="#partners">Partners</a></li>
+					<li><a href="#gallery">Gallery</a></li>
+					<li><a href="#guide">Production guide</a></li>
+					<li><a href="#team">Who's behind it</a></li>
+					<li><a href="#contact">Contact</a></li>
+					<li><a href="index.html"> <span class="lang-sm lang-lbl" lang="cs"></span></a></li>
+				</ul>
+			</div>
+		</nav>
+
+		<div class="home-wrapper">
+			<div class="container">
+				<div class="row">
+
+					<div class="col-md-10 col-md-offset-1">
+						<div class="home-content">
+							<h1 class="white-text text-center">Míčkoflus</h1>
+							<p class="white-text text-center">Shoot them all!</p>
+							<a href="#about"><button class="white-btn">Get more information</button></a>
+						</div>
+					</div>
+				</div>
+			</div>
+		</div>
+	</header>
+
+	<div id="about" class="section md-padding">
+		<div class="container">
+			<div class="row">
+				<div class="section-header">
+					<h2 class="title">About Míčkoflusu</h2>
+					<p>It drives, shoots it and looks innocent. This is Míčkoflus &#8212;
+					robotics kit from Robotárna (department of <a target="_tab" href="https://helceletka.cz">
+					Youth Centre Helceletova Brno</a>). This kit was firstly introduce on <a href="">RoboticCamp 2018</a>.</p>
+				</div>
+
+				<div class="row row-eq-height">
+					<div class="col-md-4 xsm-padding">
+						<div class="about">
+							<i class="fa fa-tree"></i>
+							<h3>Unmistakable wooden design</h3>
+							<p>Strong, light, easy glueable. All thanks to wood.</p>
+						</div>
+					</div>
+
+					<div class="col-md-4 xsm-padding">
+						<div class="about">
+							<i class="fa fa-cogs"></i>
+							<h3>5 independent motors</h3>
+							<p>Thanks to it, Míčkoflus can ride, aim and shoot. Encoders can be connected to the motors and controlled precisely.</p>
```

Comment:
Tanks se jako označení původu nepoužívá. Je potřeba použít asi opis -> The motors allow Míčkoflus to ride, aim and shoot.

You can add encoders to the motors and therefore, control them more smoothly/precisely.